macros/translations.sql

```diff
@@ -0,0 +1,11 @@
+{%- macro translate_string(string_id, default_column_name=null) -%}
+    {%- set language = var('language') -%}
+    {%- set query -%}
+        select text from {{ source('tamanu', 'translated_strings') }}
+        where string_id= '{{ string_id }}' and language= '{{ language }}'
+    {%- endset -%}
+    {%- set result = run_query(query) -%}
+    {%- if execute -%}
+        {{- (result.columns[0].values()[0] if result.rows | length > 0 else default_column_name) | trim -}}
```

Comment:
1. I am not a fan of using `trim` as it hides data entry errors which can be more difficult to debug down the track.
2. I have a preference to use the format if statement else statement for readability.
```suggestion
        {{- (result.columns[0].values()[0] if result.rows | length > 0 else default_column_name) -}}
```